CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 15-5.6 - 2025-05-20
+* Change storage backup retention policy
+* 7 daily backups, 4 weekly, 3 monthly, 2 yearly
+* Remove file backup snapshots we can't decrypt
+* Reduce huge file backup retention
```

Comment:
I think all points above (except `Remove file backup snapshots`) are the same single point really: Reduce huge file backup retention.

Reply:
Updated.